ipatests/pytest_ipa/deprecated_frameworks.py

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (C) 2019  FreeIPA Contributors see COPYING for license
+#
+
+"""
+TL;DR
+Warns about xunit/unittest/nose tests.
+
+FreeIPA is a rather old project and hereby includes all the most
+famous in the past and present Python test idioms. Of course,
+this is difficult to play around all of them. For now, the runner
+of the IPA's test suite is Pytest.
+
+Pytest supports xunit style setups, unittest, nose tests. But this
+support is limited and may be dropped in the future releases.
+Worst of all is that the mixing of various test frameworks results
+in weird conflicts and of course, is not widely tested. In other
+words, there is a big risk. To eliminate this risk and to not pin
+Pytest to 3.x branch IPA's tests were refactored.
+
+This plugin is intended to issue warnings on collecting tests,
+which employ unittest/nose frameworks or xunit style.
+
+To treat these warnings as errors it's enough to run Pytest with:
+
+-W error:'xunit style is deprecated':pytest.PytestDeprecationWarning
+
+"""
+import sys
+import pytest
+
+forb_mod_scope = [
```

Comment:
What's forb? Forbidden? Please spell it out as ``forbidden_module_scopes`` and ``forbidden_class_scopes``